README.md

```diff
@@ -22,12 +25,28 @@ Sarek can also handle tumour / normal pairs and could include additional relapse
 
 The pipeline is built using [`Nextflow`](https://www.nextflow.io), a workflow tool to run tasks across multiple compute infrastructures in a very portable manner. It comes with `Docker` containers making installation trivial and results highly reproducible.
```

Comment:
It comes with 'Docker'... is this still up-to date or should we add the other container engines, since we don't have a monolithic Docker file anymore?

Reply:
Good catch, we might need to update the TEMPLATE

Reply:
-> https://github.com/nf-core/tools/pull/850

Reply:
that was fast 🚒